web/packages/teleport/src/test/helpers/bots.ts

```diff
@@ -72,17 +74,22 @@ export const getBotSuccess = (overrides?: {
  * @returns http handler to use in SetupServerApi.use()
  */
 export const editBotSuccess = (
-  version: 1 | 2 = 2,
+  version: 'v1' | 'v2' | 'v3' = 'v3',
```

Comment:
Seems like this type is used a few types, maybe `export type EditBotApiVersion`?